string uuid
string which_qsr
string episodes
string[] objects
string iGraph
activity_data/Graphlets histogram
int32 number_of_detections
string map_name
string current_node
time start_time
time end_time
